backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/AnnouncementService.java

```diff
@@ -186,20 +185,7 @@ private void deleteAnnouncement(NormExpressionEli expressionEli) {
       deleteAnnouncementByNormEliPort.deleteAnnouncementByNormEli(
         new DeleteAnnouncementByNormEliPort.Command(expressionEli)
       );
-      announcement
-        .get()
-        .getReleases()
-        .stream()
-        .map(Release::getPublishedNorms)
-        .flatMap(Collection::stream)
-        .forEach(norm ->
-          deleteNormPort.deleteNorm(
-            new DeleteNormPort.Command(
-              norm.getManifestationEli(),
-              NormPublishState.QUEUED_FOR_PUBLISH
-            )
-          )
-        );
+      // TODO: (Malte Laukötter, 2025-02-25) we originally deleted all releases and published norms from the announcement here as well should we do this again?
```

Comment:
this is still here as it will be touched again when removing the delta stuff from the upload